### ISSUE TITLE
Unintuitive setting of route prefixes

### ISSUE THREAD

**Issue by lbadura** [ThoughtWorksStudios/oauth2_provider#3]
To set a route prefix for oauth related URIs environment variables need to be set like:

```
ENV['ADMIN_OAUTH_URL_PREFIX'] = 'admin/'
ENV['USER_OAUTH_URL_PREFIX'] = '/admin'
```

Since those variables are applied to different things ("map.connect" , ":as" option for "map.resources") we have two variants of '/' placement. Perhaps a helper method to set those options would be a bit more intuitive ?

**Comment by djrice** [comment]
Hi - We agree that this is confusing.  We'll try to push a fix ASAP.

**Comment by ketan** [comment]
We've pushed a fix for this issue in commit fc70f282.

routes.rb is now resilient to any leading or trailing slashes.

**Comment by lbadura** [comment]
Thanks !
